mongoimport/mongoimport.go

```diff
@@ -556,11 +555,11 @@ func (up *upserter) Insert(doc interface{}) error {
 	var err error
 	if selector == nil { // modeInsert || doc-not-exist
 		log.Logvf(log.Info, "Could not construct selector from %v, falling back to insert mode", up.imp.upsertFields)
-		err = up.collection.Insert(document)
+		_, err = up.collection.InsertOne(nil, document, driverOpts.InsertOne().SetBypassDocumentValidation(up.bypassDocValidation))
```

Comment:
What about caching the options (for here and below) so we're not alloc'ing it fresh for every operation?  I assume we'd need to cache on the inserter/upserter struct.  Or possibly that struct could have a closure generated in the constructor for doing the insertion depending on mode with the right options and this Insert method just dispatches to the closure.

Reply:
Done.